### PR TITLE
docs(curriculum): Fixed phrase that reads, "Let's add some CSS," which goes against the curriculum's requirement that the OP do the task on his or her own.

### LINKED ISSUE #48011: Line goes against our challenge document in building a registration form project
### Describe the Issue

In step 8 of building a registration form project, there is a text that go against [challenge document](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions)

> To spruce the project up, let us add some CSS.

This should be changed to 

> To spruce the project up, add some CSS.

Here is the markdown file https://github.com/freeCodeCamp/freeCodeCamp/blob/1459bc6879ce700830e710f30764b4562f2f52d3/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md#L11

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-8

### Additional context

Please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

Sometimes we may get more than one pull requests. We typically accept the most quality contribution, followed by the one that is made first.

Happy contributing.

### PR DESCRIPTION
**Pull request by KeshariPiyush24**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48011

### CODE REVIEW

**Comment by PiyushKeshari24** [comment]
This PR will address #48011

**Comment by jeremylt** [comment]
As a new contributor, we encourage you to read our contributing guidelines.

We expect our contributors to be aware of the process specific to this project. Following the guidelines religiously earns you the respect of fellow maintainers and saves everyone time.

Some examples of this are:

 1) Do not edit off of GitHub–while you can, it's not a good idea.
 2) Make sure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously.
 3) Use the correct way to link PRs in the description of the PR by updating the XXXXXX. Do not just add issue numbers everywhere and anywhere you feel like.
 4) Use mentions in comments or review requests to a minimum or not. We understand you are excited about contributing, and our maintainers will get back to you as soon as they get a chance.

We appreciate you taking the time to help us, and we hope to see more contributions from you.

Happy Contributing.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ dashedName: step-8
 # --description--
 The `vh` unit stands for viewport height, and is relative to 1% of the `height` of the viewport.
 
-To spruce the project up, let us add some CSS. Begin by giving the `body` a `width` of `100%`, and a `height` of `100vh`.
+To spruce the project up, add some CSS. Begin by giving the `body` a `width` of `100%`, and a `height` of `100vh`.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md:11]: We can de-yoda this sentence at the same time:
```suggestion
It is time to spruce the project up with some CSS. Begin by giving the `body` a `width` of `100%`, and a `height` of `100vh`.
```

**Comment by PiyushKeshari24** [comment]
Closed this PR because source branch is main branch. I have created another PR(#48017) for this.

**Pull request closed without merging**